packages/babel-node/test/index.js

```diff
@@ -1,200 +1,22 @@
-
-  return function (callback) {
-    saveInFiles(opts.inFiles);
-    let args = [binLoc];
-    args.push("--config-file", "../config.json");
-    args = args.concat(opts.args);
-
-    const spawn = child.spawn(process.execPath, args);
-
-    let stderr = "";
-    let stdout = "";
+const binLoc = path.join(dirname, "../bin/babel-node.js");
+const childLoc = path.join(dirname, "fixtures/misc/child.js");
 
-    spawn.stderr.on("data", function (chunk) {
-      stderr += chunk;
-    });
-
-    spawn.stdout.on("data", function (chunk) {
-      stdout += chunk;
-    });
-
-    spawn.on("close", function () {
-      let err;
-
-      try {
-        assertTest(stdout, stderr, opts);
-      } catch (e) {
-        err = e;
-      }
-
-      if (err) {
-        err.message =
-          args.map(arg => `"${arg}"`).join(" ") + ": " + err.message;
-      }
-
-      callback(err);
-    });
-
-    if (opts.stdin) {
-      spawn.stdin.write(opts.stdin);
-      spawn.stdin.end();
-    }
-  };
-};
-
-fs.readdirSync(fixtureLoc).forEach(function (binName) {
-  if (binName[0] === ".") return;
-
-  const suiteLoc = path.join(fixtureLoc, binName);
-  describe("bin/" + binName, function () {
-    let cwd;
+describe("babel-node", () => {
+  it("ipc works with spawned babel-node process", done => {
+    expect.assertions(1);
 
-    beforeEach(() => {
-      cwd = process.cwd();
-
-      if (fs.existsSync(tmpLoc)) {
-        for (const child of fs.readdirSync(tmpLoc)) {
-          rimraf.sync(path.join(tmpLoc, child));
-        }
-      } else {
-        fs.mkdirSync(tmpLoc);
-      }
-
-      process.chdir(tmpLoc);
-    });
-
-    afterEach(() => {
-      process.chdir(cwd);
+    const child = spawn(binLoc, [childLoc], {
+      stdio: ["inherit", "inherit", "inherit", "ipc"],
     });
 
-    fs.readdirSync(suiteLoc).forEach(function (testName) {
-      if (testName[0] === ".") return;
-
-      const testLoc = path.join(suiteLoc, testName);
-
-      const opts = {
-        args: [],
-      };
-
-      const optionsLoc = path.join(testLoc, "options.json");
-      if (fs.existsSync(optionsLoc)) Object.assign(opts, require(optionsLoc));
-
-      ["stdout", "stdin", "stderr"].forEach(function (key) {
-        const loc = path.join(testLoc, key + ".txt");
-        if (fs.existsSync(loc)) {
-          opts[key] = helper.readFile(loc);
-        } else {
-          opts[key] = opts[key] || "";
-        }
-      });
-
-      opts.outFiles = readDir(path.join(testLoc, "out-files"), fileFilter);
-      opts.inFiles = readDir(path.join(testLoc, "in-files"), fileFilter);
-
-      const babelrcLoc = path.join(testLoc, ".babelrc");
-      if (fs.existsSync(babelrcLoc)) {
-        // copy .babelrc file to tmp directory
-        opts.inFiles[".babelrc"] = helper.readFile(babelrcLoc);
-      }
-
-      it(testName, buildTest(binName, testName, opts), 20000);
+    child.on("message", msg => {
+      expect(msg).toBe("hello");
+      done();
     });
-  });
+  }, /* timeout */ 500);
```

Comment:
```suggestion
  }, /* timeout */ 20_000);
```

It's the same as all the other `babel-node` tests.